aggsender/validator/cert_hash.go

```diff
@@ -37,3 +33,17 @@ func HashCertificateToSign(cert *agglayertypes.Certificate) (common.Hash, error)
 		cert.CertificateID().Bytes(),
 	), nil
 }
+
+func getAggchainParams(cert *agglayertypes.Certificate) []byte {
```

Comment:
Nitpick: add a comment to the function

Reply:
Fix: https://github.com/agglayer/aggkit/pull/954/commits/47b0bc29164f25a6754211ab6b46a92b4330bc69